plugins/catalog/src/components/EntitySwitch/conditions.ts

```diff
@@ -14,11 +14,12 @@
  * limitations under the License.
  */
 
-import {
-  ComponentEntity,
-  ResourceEntity,
-  Entity,
-} from '@backstage/catalog-model';
+import { Entity } from '@backstage/catalog-model';
+
+export interface EntityPredicates {
+  kind: string | string[];
```

Comment:
```suggestion
  kind?: string | string[];
```

I think we should make this optional as well to open up for even more flexibility in the future.